### PR TITLE
DEV: Restrict genie package to use python 3

### PR DESCRIPTION
**Pull request by thomasyu888**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,18 +10,7 @@ before_install:
 
 matrix:
   include:
-    - os: linux
-      python: 2.7.14
     - os: linux
       python: 3.5.3
     - os: linux
       python: 3.6.2
```
Comment (kdaily) [inline .travis.yml:16]: Why not Python 3.7?

File: setup.py [hunk 2]
```diff
@@ -9,11 +9,12 @@
       license='MIT',
       packages=find_packages(),
       zip_safe=False,
+      python_requires='>=3.0',
```
Comment (kdaily) [inline setup.py]: This should be at least 3.5 since the travis build starts there.

File: setup.py [hunk 3]
```diff
@@ -9,11 +9,12 @@
...
       entry_points={
         'console_scripts': ['genie = genie.__main__:main']},
       install_requires=[
         'pandas>=0.20.0',
         'synapseclient>=1.9',
         'httplib2>=0.11.3',
         'pycrypto>=2.6.1',
-        'PyYAML>=3.11'])
+        'PyYAML>=5.1'])
```
Comment (kdaily) [inline setup.py:20]: why such a big jump?

**Review by kdaily — CHANGES_REQUESTED** [review]

File: setup.py (hunk 3 above)
Comment (thomasyu888) [reply setup.py:20]: I was just using an old pyyaml version.  There are new specs about how to load yaml file which made me have to use `5.1` #78

File: .travis.yml (hunk 1 above)
Comment (thomasyu888) [reply .travis.yml:16]: Added.

**Comment by thomasyu888** [comment]
@kdaily Fixed.

**Pull request merged**